playground/nextjs/pages/index.tsx

```diff
@@ -18,6 +19,10 @@ const Home: NextPage = () => {
         />
       </Head>
 
+      <SignedOut>
+        <__experimental_GoogleOneTap />
+      </SignedOut>
```

Comment:
❓ do we want to put a signed out check inside the component itself?

Reply:
We do have that check inside `one-tap-start.tsx`. I will be dropping this file change tho.